balancer_conn_wrappers.go

```diff
@@ -205,36 +117,24 @@ func (ccb *ccBalancerWrapper) updateSubConnState(sc balancer.SubConn, s connecti
 	if sc == nil {
```

Comment:
Optional: instead of this case and a big huge comment explaining it, can we suppress the updateSubConnState call, possibly?

Reply:
Looked at it for a little while. I think it would need some work to make that happen. Added a todo for it. But unfortunately, that makes the comment even longer.